tests/test_logging.py

```diff
@@ -64,6 +64,7 @@ def setUp(self):
         self.logdir = tempfile.mkdtemp(prefix="dnf-logtest-")
         self.log_size = 1024 * 1024
         self.log_rotate = 4
+        self.log_compress = False
```

Comment:
Why not add a test for the compression here? Not possible for some reason?

Reply:
I'll look into it.

Reply:
I added a test.